h2o-docs/src/product/data-science/stacked-ensembles.rst

```diff
@@ -103,8 +103,10 @@ Defining a Stacked Ensemble Model
 
 You can follow the progress of H2O's Stacked Ensemble development `here <https://0xdata.atlassian.net/issues/?filter=19301>`__.
 
-Example
-~~~~~~~
+Examples
+~~~~~~~~
+
+Below is a simple example showing how to build a Stacked Ensembles model.
```

Comment:
The Stacked Ensembles import a dataset from Erin's s3 bucket. It might be time to point these to the small data folder instead. Specifically:
- For train: ("https://s3.amazonaws.com/h2o-public-test-data/smalldata/testng/higgs_train_5k.csv")
- For test: ("https://s3.amazonaws.com/h2o-public-test-data/smalldata/testng/higgs_test_5k.csv")

(and yes, the folder is "testng" and not "testing" :halo:)

Reply:
Should I also update the User Guide Stacked Ensembles example for this as a tag-along fix?